src/main/java/org/nustaq/serialization/FSTSerializerRegistry.java

```diff
@@ -108,11 +108,11 @@ final FSTObjectSerializer getSerializer(Class cl, Class lookupStart) {
         }
         final SerEntry serEntry = map.get(cl);
         if ( serEntry != null ) {
-            if ( cl == lookupStart && serEntry.ser.willHandleClass(cl)) {
+            if ( cl == lookupStart && serEntry.ser.willHandleClass(lookupStart)) {
```

Comment:
as `cl` and `lookupStart` are equal by reference, there is no difference which one use pass as an argument.

Reply:
Indeed, in the first if-clause `cl` could be used since it is equal to `lookupStart` in that clause, I changed it there as well to harmonize it with the second if-clause (so that a casual reader won't think the difference might be important).